app.rb

```diff
@@ -42,19 +42,7 @@ def create_person
     age = gets.chomp
     print 'Name: '
     name = gets.chomp
```

Comment:
- [ ] Good job so far 👍 but all requests for input and input validation should've been placed in a different file whose responsibilities are to request or validate input. Kindly correct the `create_person` method and make it more like the `list_people` or `create_book` methods as those have a single responsibility 👍 . You can read more on that [here](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid`).